fake.py

```diff
@@ -413,6 +422,8 @@ def parse_args():
                         help="Add a hologram effect")
     parser.add_argument("--no-ondemand", action="store_true",
                         help="Continue processing when no consumers are present")
+    parser.add_argument("--background-mask-update-speed", default="10", type=int,
```

Comment:
I don't like the lag with the default update speed. Should we make it go faster by default?

Reply:
what's the default you are thinking to?

Reply:
30 to 50 maybe?

Reply:
ok